### PR TITLE
Floydwarshall Algorithm in c++

### PR DESCRIPTION
**Pull request by Nishtha3512**
Shortest path Algorithm issue #159

### CODE REVIEW

**Comment by Nishtha3512** [comment]
issue #159  Floydwarshal Algorithm
for example 
floyd

**Comment by SakshiShreya** [comment]
Please correct the PR template and mention the issue number in PR template.

File: Floydwarshall Algorithm in c++ [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <conio.h>
+using namespace std;
+void floydwarshal(int cost[4][4])
```
Comment (SakshiShreya) [inline Floydwarshall Algorithm in c++]: Add a new-line after line 2.

File: Floydwarshall Algorithm in c++ [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+{
+    int i, j, k;
+    for (k = 0; k < 4; k++)
+    {
+        for (i = 0; i < 4; i++)
+        {
+            for (j = 0; j < 4; j++)
+            {
+                  if ((cost[i][k] + cost[k][j] < cost[i][j]) )
```
Comment (SakshiShreya) [inline Floydwarshall Algorithm in c++]: Remove the extra pair of parentheses.

File: Floydwarshall Algorithm in c++ [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                    {
+                        cost[i][j] = cost[i][k] + cost[k][j];  //change only if value is less
+                    }
+            }
+        }
+    }
+    for (i = 0; i < 4; i++)
+    {
+        cout<<"\nMinimum Cost With Respect to Node:"<<i<<endl;
```
Comment (SakshiShreya) [inline Floydwarshall Algorithm in c++]: Add a space around `<<`

File: Floydwarshall Algorithm in c++ [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        for (j = 0; j < 4; j++)
+        {
+            cout<<cost[i][j]<<"\t";
+        }
+    }
+}
+int main()
```
Comment (SakshiShreya) [inline Floydwarshall Algorithm in c++]: There should be a new line between 2 functions.

File: Floydwarshall Algorithm in c++ [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        cout<<"enter values for "<<(i+1)<<" row"<<endl;
+        for (int j = 0; j < 4; j++)
+        {
+            cin>>cost[i][j];
+        }
+    }
+    floydwarshal(cost);
+    getch();
+}
```
Comment (SakshiShreya) [inline Floydwarshall Algorithm in c++]: Add a sample input and output at EOF

**Comment by somya-kapoor** [comment]
@Nishtha3512 Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than a 7 days, I am closing this PR.

**Pull request closed without merging**